src/top/vision/region.h

```diff
@@ -0,0 +1,91 @@
+/*!
```

Comment:
should we use `src/top/vision/yolo2/`? @tqchen

Reply:
If this op is yolo2 specific, then we should do that